src/types/announcements.ts

```diff
@@ -0,0 +1,14 @@
+// NOTE: Only `New Feature` for now but we can extend this for eg: `Bugfix` or `Rework`
+export type TagVariant = "New Feature"
+
+export type Announcement = {
+  title: string
+  description: string
+  image: (props: React.SVGProps<SVGSVGElement>) => JSX.Element
+  tags: TagVariant[]
+}
+
+export type Announcements = {
```

Comment:
Hmm am abit confused here. So each announcement `batch` is a type of `Announcements` and each announcement `batch` has list of possible tabs called `announcements`? 
If this the case, shouldnt `link` be an attribute of `Announcement` rather than `Announcements`? since it could be possible for 2 announcements in a particular announcement batch to have 2 seperate links?

Reply:
to give this in context of figma, 
<img width="720" alt="Screenshot 2023-02-20 at 2 31 23 PM" src="https://user-images.githubusercontent.com/44049504/220029723-eeb97324-89d8-4886-8775-74d62544b973.png">

each `Announcements` is a modal, and each slide on that modal is an `Announcement`; wrt `link`, i clarified w/ @NatMaeTan prior and the link should be tied to each release (ie, each `Announcements`). but yea i'm open to this and in fact i think a better flow would be for each slide to have a link to the relevant section in the patch note and having an ending slide for the release overall that links to the main patch notes

Reply:
right, got it. Let's go with design then, and revisit this if we have a use case of `link per slide`

Reply:
Nit: could we rename `Announcements` to `AnnouncementBatch` ?
just feels `Announcements.announcements` could be confusing :(

Reply:
Yea that's a good idea, will do!